docsource/functionalRequirements.rst

```diff
@@ -1,2 +1,24 @@
 Functional Requirements
-========================
\ No newline at end of file
+========================
+
+1. Support the indexing of many data types (Genome, Assembly, Narrative etc). It is expected that there will be about 100 types
```

Comment:
Probably more along the lines of 1000 for order of magnitude, we already have > 100 types, not including subtypes.

Reply:
Changed.